scipy/stats/_continuous_distns.py

```diff
@@ -6407,6 +6659,9 @@ def _argcheck(self, skew):
         # for all skew args.
         return np.ones(np.shape(skew), dtype=bool)
```

Comment:
```suggestion
        return np.isfinite(skew)
```
As you did below?

Reply:
inf leads to nans, so I changed it as you proposed